library/src/main/java/com/squareup/timessquare/DefaultDayViewAdapter.java

```diff
@@ -0,0 +1,20 @@
+package com.squareup.timessquare;
+
+import android.view.ContextThemeWrapper;
+import android.widget.TextView;
+
+/**
+ * Created by Michal on 30/11/15.
+ */
+public class DefaultDayViewAdapter implements DayViewAdapter {
+
```

Comment:
again, two-char indent, no blank lines, either add a useful class comment or just remove it entirely.